src/preferences.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self'; script-src 'unsafe-inline'">
+  <link rel="stylesheet" href="./assets/css/style.min.css">
+</head>
+
+<body>
+
+  <main>
+
+    <!-- GitHub Username -->
+    <div class="field">
+      <label class="label" for="github-username">GitHub Username</label>
+      <div class="control">
+        <input id="github-username" type="text" class="input" placeholder="Type your GitHub username">
```

Comment:
It would look nice if we used Font Awesome icons for our inputs.

<img width="358" alt="screenshot 2018-11-02 at 09 09 24" src="https://user-images.githubusercontent.com/8133259/47905577-5995c300-de7f-11e8-971a-5a494ef1e021.png">

We could use the GitHub brand icon on the left and the tick or cross icon on the right depending whether the user exists or not. What do you think? 😄

Maybe for the GitHub Sync Interval field we could use a clock or something like that.

Reply:
> This is a good idea but I don't think it's required - when thinking about other desktop applications they generally save without displaying anything to the user, only web applications tend to show saving information.

You told me this when I wanted to add the check icon haha, Do I do it anyway ?

Reply:
I wrote : 
>  Add a way to tell the user the new value entered is saved like <i class="fas fa-check"></i>

Haha it's okay

Reply:
Yeah but we're checking whether the value entered is a valid username, not whether it saved 😊

Reply:
It makes sense, I will do that 👍

Reply:
Fixing the clock icon not showing on hover.
![preview.png](https://puu.sh/BV5NY/01d19489cb.png)

Reply:
Can't fix it, Can you give it a try later ?

Reply:
Sorry for the delay ! Thank's it's fixed now 🎉